### PR TITLE
HW04 Updated with solutions

### PR DESCRIPTION
**Pull request by malavikasrinivasan**
Sorry about the tardiness.

### CODE REVIEW

File: D04_ex00.py [hunk 1]
```diff
@@ -11,19 +11,55 @@
...
+        print("Bingo! You got it!")
+        return True
 
-    print("Hello World!") # Remove this and replace with your function calls
-    
+
+###############################################################################
+
+
+def main():
```
Comment (danielsgriffin) [inline D04_ex00.py]: Try to only use main to call other functions, have the distinct tasks separated into functions.

File: HW04_ch07_ex02.py [hunk 2]
```diff
@@ -26,10 +26,23 @@
 # Body
 
 
+def eval_loop():
```
Comment (danielsgriffin) [inline HW04_ch07_ex02.py]: "continue until the user enters 'done', and then return the value of the last expression it evaluated"

**Comment by danielsgriffin** [comment]
You are still uploading .DS_Store.

Good job.

**Comment by siahmed** [comment]
Hi Malavika!

File: D04_ex00.py (hunk 1 above)
Comment (meghanamurthysn) [inline D04_ex00.py]: What if the number entered by the user is not in the range of 1 to 25?

**Pull request closed without merging**